examples/routing_service/routing_service_with_security/dds_security_01/cert/D2_DP.pem

```diff
@@ -0,0 +1,70 @@
+Certificate:
```

Comment:
We should review the names of the generated certificates and keys and come up with a naming convention.

Reply:
use reference certificates.